aws-shared-1/main.tf

```diff
@@ -97,6 +97,28 @@ module "aws_az_1e" {
   workers_org_subnet_cidr = "${var.workers_org_subnet_1e_cidr}"
 }
 
+resource "aws_route53_record" "workers_org_nat" {
+  zone_id = "${var.travisci_net_external_zone_id}"
+  name = "workers-nat-org-${var.env}-${var.index}.aws-us-east-1.travisci.net"
+  type = "A"
+  ttl = "300"
+  records = [
```

Comment:
I'm guessing by the code from which I'm copying that we mostly don't care, but I also like the idea of pre-AZ records in addition to these.